.gitignore

```diff
@@ -360,4 +360,5 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
+/Wheelingful/Wheelingful.http
```

Comment:
as you deleted the file, no need to ad dit to gitignore

Reply:
Good point.